### ISSUE TITLE
Does this approach not even overfit on noisy data?

### ISSUE THREAD

**Issue by rassaire** [mobaidoctor/med-ddpm#29]
Hello,

I am using this approach for CT to pseudo-MRI conversion. In a previous comment, you mentioned that this method is sensitive to noise. I would like to know if this sensitivity is observed only in test data, or if the model fails to overfit even with noisy training data.

 My CT datasets are noisy, but I would at least like to see the model overfit the data before working on generalization. The model has been trained for 80,000 steps so far, but it has not yet overfitted. I also remember  you mentioning that you managed to achieve MRI to CT translation after exceeding 1,000,000/2,000, 000 steps.

**Comment by mobaidoctor** [comment]
Hi, @rassaire Thank you for your interest in our work. We apologize for the delayed response, we have been fully occupied with research tasks. One of the limitations we are currently addressing for our next version, Med-DDPM v2.0, is the model's inability to converge when the dataset is noisy. Our current method is highly sensitive to image quality, so if the dataset contains images with artifacts and noise, the model struggles to converge effectively. Therefore, we highly recommend using our method for its intended purpose of mask-to-image synthesis, rather than for broad image-to-image translation tasks like MRI to CT. These tasks are challenging to converge quickly, especially with poor quality images. Alternatively, you may want to wait for the next version of Med-DDPM, which will address these issues. Thank you.

**Comment by rassaire** [comment]
thanks @mobaidoctor
